spec.md

```diff
@@ -1441,6 +1441,11 @@ message ListVolumesRequest {
   // next page of entries. This field is OPTIONAL.
   // An empty string is equal to an unspecified field value.
   string starting_token = 2;
+
+  // Secrets required by plugin to complete volumes listing request.
+  // This field is OPTIONAL. Refer to the `Secrets Requirements` section
+  // on how to use this field.
+  map<string, string> secrets = 3 [(csi_secret) = true];
```

Comment:
We want to make sure all new fields, methods, etc. go through alpha/beta/ga.

So please add (alpha_field) = true to this field to indicate that it is alpha in this release, in the next release we can move to beta, and GA after that.

Reply:
See https://github.com/container-storage-interface/spec/pull/457 for example.